### ISSUE TITLE
Duplicated touch events and small fonts

### ISSUE THREAD

**Issue by official-jkim** [Freeyourgadget/Gadgetbridge#231]
1. On 'Your Activity' screen for Mi Band, sliding left and right performs both switching tabs and moving activity/sleep graphs horizontally. When you slide left or right the graph area, sometimes it switches tabs and sometimes it moves graph. Instead of using slidable tab to switch between four different graphs, it will be better to use buttons, menu or navigation drawer to switch tabs that doesn't involve horizontal sliding.

2. I'm currently using a phone with 2K display, 540 dpi. The strings in 'Your Activity' screens are too small to read well, especially one in the sleep graph. Better resizing is needed.

'Your Activity' feature is on Alpha status so it's not a big deal but it needs to be fixed.

**Comment by danielegobbetti** [comment]
Please provide a patch or make pull request for the second part of your request, as you are the only one who can verify any change related to your phone display.

**Comment by cpfeiffer** [comment]
You're right about the touch thing. One really has to know how to use it. There are buttons for switching tabs, btw, see the very bottom of the activity.
